apps/web/components/booking/BookingListItem.tsx

```diff
@@ -94,6 +94,12 @@ function BookingListItem(booking: BookingItemProps) {
     mutation.mutate(body);
   };
 
+  const getSeatReferenceUId = () => {
+    if (booking.seatsReferences.length > 0) {
+      return booking.seatsReferences[0].referenceUId;
+    }
+  };
```

Comment:
This has an implicit `return undefined` should we make it explicit?

```suggestion
  const getSeatReferenceUId = () => {
    if (booking.seatsReferences.length < 1) return undefined;
    return booking.seatsReferences[0].referenceUId;
  };
```

Reply:
I prefer just `return` other than that makes sense.

Reply:
```javascript
return booking.seatsReferences[0]?.referenceUId; // remove the need for the if altogether.
```